src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+  margin-top: 20px;
+}
+
+.form-search-big,
+.form-search-small {
+  background: #fff;
+}
+
+#input-search-big,
+#input-search-small {
```

Comment:
don't styling id, better to use classes for this

Reply:
changed to classes